README.md

```diff
@@ -1869,164 +1870,186 @@ These encoders/layouts make use of an internal buffer to hold the JSON output du
+      <td><p>Event timestamp.</p>
         <ul>
-          <li><tt>fieldName</tt> - Output field name (<tt>level</tt>)</li>          
+          <li><tt>fieldName</tt> - Output field name (<tt>@timestamp</tt>)</li>
+          <li><tt>pattern</tt> - Output format (<tt>[ISO_OFFSET_DATE_TIME]</tt>)  See <a href="#customizing-timestamp">above</a> for possible values.</li>
+          <li><tt>timeZone</tt> - Timezone (local timezone)</li>
         </ul>
       </td>
     </tr>
     <tr>
-      <td><tt>logLevelValue</tt></td>
-      <td><p>Logger level numerical value </p>
+      <td><tt>uuid</tt></td>
+      <td>
+        Outputs random UUID as field value. Handy when you want to provide unique identifier
+        for log lines.
+      
         <ul>
-          <li><tt>fieldName</tt> - Output field name (<tt>level_value</tt>)</li>
+          <li><tt>fieldName</tt> - Output field name (<tt>uuid</tt>)</li>
+          <li><tt>strategy</tt> - UUID generation strategy (<tt>random</tt>). Supported options: <ul><li><tt>random</tt> - for Type 4 UUID</li>
+          <li><tt>time</tt> - for Type 1 time based UUID</li>
+          </ul></li>
+          <li><tt>ethernet</tt> - Only for 'time' strategy. When defined - MAC address to use for location part of UUID. Set it to <tt>interface</tt> value to use real underlying network interface or to specific values like <tt>00:C0:F0:3D:5B:7C</tt></li>          
         </ul>
+	      <p>Note: The <a href="https://mvnrepository.com/artifact/com.fasterxml.uuid/java-uuid-generator/"><tt>com.fasterxml.uuid:java-uuid-generator</tt></a> optional dependency must be added to applications that use the `uuid` provider.</p>
       </td>
     </tr>
     <tr>
-      <td><tt>callerData</tt></td>
-      <td><p>Outputs data about from where the logger was called (class/method/file/line)
+      <td><tt>version</tt></td>
+      <td><p>Logstash JSON format version.</p>
         <ul>
-          <li><tt>fieldName</tt> - Sub-object field name (no sub-object)</li>
-          <li><tt>classFieldName</tt> - Field name for class name (<tt>caller_class_name</tt>)</li>
-          <li><tt>methodFieldName</tt> - Field name for method name (<tt>caller_method_name</tt>)</li>
-          <li><tt>fileFieldName</tt> - Field name for file name (<tt>caller_file_name</tt>)</li>
-          <li><tt>lineFieldName</tt> - Field name for line number (<tt>caller_line_number</tt>)</li>
+          <li><tt>fieldName</tt> - Output field name (<tt>@version</tt>)</li>
+          <li><tt>version</tt> - Output value (<tt>1</tt>)</li>
+          <li><tt>writeAsInteger</tt> - Write the version as a integer value (<tt>false</tt> = write as a string value)</li>
         </ul>
       </td>
     </tr>
+  </tbody>
+</table>
+
+
+### Providers for LoggingEvents
+
+The table below lists the available providers for _LoggingEvents_, and their configuration properties (defaults in parentheses).
+The provider name is the xml element name to use when configuring.
```

Comment:
Mention and provide a link to the common providers section somewhere in this paragraph.

Perhaps:

```suggestion
The [common providers mentioned above](#providers-common-to-loggingevents-and-accessevents), and the providers listed in the table below, are available for _LoggingEvents_.
The provider name is the xml element name to use when configuring.  Each provider's configuration properties are shown, with default configuration values in parenthesis.
```